test/configuration-tests/with-networks/check.ts

```diff
@@ -44,10 +44,10 @@ execution = hardhatStarknetTest("--no-compile test/contract-factory-test.ts".spl
 assertContains(execution.stderr, expected);
 console.log("Success");
 
-console.log("Testing with alpha-goerli2 config network");
-process.env.NETWORK = "alpha-goerli2";
-hardhatStarknetDeploy(
-    "starknet-artifacts/contracts/contract.cairo --starknet-network alpha-goerli2 --inputs 10".split(
-        " "
-    )
-);
+// console.log("Testing with alpha-goerli2 config network");
```

Comment:
Suppression of this should be documented in development related changes in the PR description. Also, a message can be logged here about the suppression.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/264/commits/adc84b2a14c45a350409b4a31486ad57f2cfd08d